tests/unit/test_context.py

```diff
@@ -285,10 +285,10 @@ async def test_pipeline_matrix(batch_config_loader: ConfigLoader) -> None:
     )
 
     assert ctx.graph == {
-        ("task-1-e3-o3-t3",): set(),
-        ("task-1-o1-t1",): set(),
-        ("task-1-o2-t1",): set(),
-        ("task-1-o2-t2",): set(),
+        "task-1-e3-o3-t3": set(),
```

Comment:
Does it change `neuro-flow inspect <bake> --view` functionality?
Is `FullID` -> `str` replacement required?

Reply:
> Does it change neuro-flow inspect <bake> --view functionality

No, it stays the same. I've added some transformation code to `BatchRunner._build_graphs`
> Is FullID -> str replacement required?

This is to make `Graph` (`GraphEmb`) much more simple (or it would had to check that all nodes in graph have same prefix). I can revert this change, but I had an idea to remove prefix from contexts in general (still not sure about all cases), as they are only needed for executor, not for expressions in yaml.

Reply:
ok